gitversioner/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'kotlin'
+apply plugin: 'com.novoda.bintray-release'
+
+group = 'com.pascalwelsch.gitversioner'
+version = '0.0.1-SNAPSHOT'
+
+buildscript {
+    ext.kotlin_version = '1.1.2-4'
+    configure([repositories, project.repositories]) {
+        maven { url 'https://repo.gradle.org/gradle/repo' }
```

Comment:
the gradle maven repo contains the latest versions. `1.1.2-2` has problems with latest stable Android Studio, `1.1.2-4` is required.